### PR TITLE
logging and cleanup

### PR DESCRIPTION
**Pull request by muhammad-ammar**

### CODE REVIEW

File: VEDA/utils.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    with open(yaml_config_file, 'r') as config:
+        try:
+            config_dict = yaml.load(config)
+        except yaml.YAMLError:
+            pass
+
+    return config_dict
+
+# TODO! This should be moved to into a separate file once settings are refactored.
+def get_logger_config(log_dir='/var/tmp',
```
Comment (Qubad786) [inline VEDA/utils.py]: Are we going to add tests for this one?

Comment (muhammad-ammar) [reply VEDA/utils.py]: nope. As settings are being refactored, so this code will be move into settings directory which  will contain all setting files for different environments.

**Comment by muhammad-ammar** [comment]
@edx/educator-mallow This is ready for review.

File: VEDA/settings.py [hunk 2]
```diff
@@ -2,15 +2,27 @@
...
 ROOT_DIR = os.path.dirname(os.path.dirname((__file__)))
 
+CONFIG_DATA = get_config()
+
+DJANGO_SECRET_KEY = CONFIG_DATA['django_secret_key']
+DJANGO_ADMIN = ('', '')
+DJANGO_DEBUG = CONFIG_DATA.get('debug', False)
+DATABASES = CONFIG_DATA.get('DATABASES')
+STATIC_ROOT_PATH = CONFIG_DATA.get(
+    'STATIC_ROOT_PATH',
```
Comment (yro) [inline VEDA/settings.py]: @muhammad-ammar This is much nicer! Thank you.

File: VEDA/settings/utils.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                'propagate': True,
+                'level': 'WARNING'
+            },
+            'factory': {
+                'handlers': handlers,
+                'propagate': True,
+                'level': 'WARNING'
+            },
+            'django.request': {
+                'handlers': handlers +  [] if debug else ['mail_admins'],
```
Comment (Qubad786) [inline VEDA/settings/utils.py]: Nit: extra space after +

**Comment by muhammad-ammar** [comment]
@muzaffaryousaf @Qubad786 @mushtaqak Please have a final look so that we can merge this as soon as possible.

File: VEDA/settings/production.py [hunk 4]
```diff
@@ -2,12 +2,15 @@
...
 from VEDA.settings.base import *
 from VEDA.utils import get_config
+from VEDA.settings.utils import get_logger_config
 
 DEBUG = False
 TEMPLATE_DEBUG = DEBUG
 
 ALLOWED_HOSTS = ['*']
 
+LOGGING = get_logger_config()
```
Comment (muzaffaryousaf) [inline VEDA/settings/production.py]: Don't we need to pass values for this ?

Comment (muzaffaryousaf) [reply VEDA/settings/production.py]: NVM. Got it.

**Comment by muzaffaryousaf** [comment]
LGTM 👍

**Comment by muzaffaryousaf** [comment]
@muhammad-ammar Let's rebase N squash the commits.

**Pull request merged**